FlowCrypt/Functionality/Services/EKMVcHelper.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  EKMVcHelper.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 4/28/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol EKMVcHelperType {
+    func refreshKeysFromEKMIfNeeded(in viewController: UIViewController)
+}
+
+final class EKMVcHelper: EKMVcHelperType {
+
+    let appContext: AppContextWithUser
+    let keyMethods: KeyMethodsType
+
+    init(appContext: AppContextWithUser) {
+        self.appContext = appContext
+        self.keyMethods = KeyMethods()
+    }
+
+    func refreshKeysFromEKMIfNeeded(in viewController: UIViewController) {
+        Task {
+            do {
+                // Sleep for 3 seconds when mock testing
```

Comment:
@sosnovsky, @tomholub So Tests should pass now.
@tomholub FYI only changes I added after your approval is this logic.
Let me know what you think.

```
if Bundle.isDebugBundleWithArgument("--mock-fes-api") {
  try await Task.sleep(nanoseconds: 3 * 1000 * 1_000_000)
}
```